atomics/T1218.003/T1218.003.yaml

```diff
@@ -49,4 +49,24 @@ atomic_tests:
     command: |
       cmstp.exe /s #{inf_file_uac} /au
     name: command_prompt
+- name: PowerShell Lateral Movement
```

Comment:
Did you mean to add this to [T1218.003 - CMSTP](https://attack.mitre.org/techniques/T1218/003/)? I see you also moved it to T1021.003 which would be the correct sub-technique.

Reply:
Fixed!